### DISCUSSION TITLE
Having vcpkg start

### DISCUSSION THREAD

**Discussion by LedLoaf** [microsoft/vcpkg#36036] [Q&A]
I've asked this before, but can't seem to find my post. I think I said to just delete it.

How do I start vcpkg in Visual Studio default as off? It's annoying when I open a new project and I get a bunch of conflicting errors because vcpkg starts on by default.

**Comment by StarGate-One** [comment]
Is this the issue being referenced https://github.com/microsoft/vcpkg/issues/26385 ?

**Reply by LedLoaf** [reply]
Sorry for the ridiculously late reply. Yes sir, thank you very much.
